crates/geometry/src/line.rs

```diff
@@ -90,21 +77,35 @@ impl<Frame, const DIMENSION: usize> Distance<Point<Frame, DIMENSION>> for Line<F
     }
 }
 
+impl<Frame, const DIMENSION: usize> Default for Line<Frame, DIMENSION> {
+    fn default() -> Self {
+        Self {
+            point: Default::default(),
+            direction: Vector::zeros(),
+        }
+    }
+}
```

Comment:
You should be able to derive that?!

Reply:
I don't know why, but Vector<Framed, DIMENSION> doesn't seem to implement Default.

Reply:
I now added a comment explaining why default can not be derived for `[f32; DIMENSION]`.